client/lib/analytics/ad-tracking/record-order.js

```diff
@@ -313,59 +319,59 @@ function recordOrderInFloodlight( cart, orderId, wpcomJetpackCartInfo ) {
  *
  * @param {object} cart - cart as `CartValue` object
  * @param {number} orderId - the order id
+ * @param {object} wpcomJetpackCartInfo - info about WPCOM and Jetpack in the cart
  * @returns {void}
  */
-function recordOrderInFacebook( cart, orderId ) {
+function recordOrderInFacebook( cart, orderId, wpcomJetpackCartInfo ) {
 	if ( ! isAdTrackingAllowed() || ! isFacebookEnabled ) {
 		return;
 	}
 
-	/**
-	 * We have made a conscious decision to ignore the 0 cost carts, such that these carts are not considered
-	 * a conversion. We will analyze the results and make a final decision on this.
-	 */
-	if ( cart.total_cost < 0.01 ) {
-		debug( 'recordOrderInFacebook: skipping due to a 0-value cart.' );
-		return;
-	}
-
 	const currentUser = getCurrentUser();
 
-	// Fire both WPCom and Jetpack pixels
-
 	// WPCom
-
-	let params = [
-		'trackSingle',
-		TRACKING_IDS.facebookInit,
-		'Purchase',
-		{
-			product_slug: cart.products.map( ( product ) => product.product_slug ).join( ', ' ),
-			value: cart.total_cost,
-			currency: cart.currency,
-			user_id: currentUser ? currentUser.hashedPii.ID : 0,
-			order_id: orderId,
-		},
-	];
-	debug( 'recordOrderInFacebook: WPCom', params );
-	window.fbq( ...params );
+	if ( wpcomJetpackCartInfo.containsWpcomProducts ) {
+		if ( null !== wpcomJetpackCartInfo.wpcomCostUSD ) {
```

Comment:
Will this produce the same result as the deleted code above that checks `cart.total_cost < 0.01`? Or is this changing the functionality to start tracking zero value purchases?

Reply:
No. Thank you for catching! That's another bug I think, taken from another part of the code. I'll update shortly.

Reply:
Resolved.